management/backup.py

```diff
@@ -317,7 +312,7 @@ def service_command(service, command, quit=None):
 		# Start services again.
 		service_command("dovecot", "start", quit=False)
 		service_command("postfix", "start", quit=False)
-		service_command("php7.2-fpm", "start", quit=False)
+		service_command("php8.0-fpm", "start", quit=False)
```

Comment:
There are some instances that have ...php8.0... and some that have php$PHP_VER, should this be unified to also use `phpPHP_VER`?